src/components/Table.web.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import { getTheme } from '../modules/theme';
+import { TableProperties } from './Table';
+import { View, Text, StyleSheet, Dimensions, FlatList } from 'react-native';
+import { ehIE } from '../modules/browser';
+
+interface State {
+  leftHeader: number;
+}
+
+export class Table extends React.Component<TableProperties, State> {
+  state: State = {
+    leftHeader: 0
+  };
+
+  render() {
+    const { columns, data, idAttribute, cellStyle } = this.props;
+
+    const { leftHeader } = this.state;
+
+    const heightContainer = Dimensions.get('window').height - 303; //Topo = 60px, padding = 16px, filtrar = 54px, filtros = 88px, padding = 16px, header = 53px, padding = 16
+
+    return (
+      <>
+        <div
+          style={{
+            overflow: 'hidden',
```

Comment:
é possível colocar os estilos que estão inline em um `StyleSheet`?
obs.: algumas vezes por estar em uma div não funciona o estilo, tomar cuidado se for alterar.